### PR TITLE
Updating the docs for submitting results

### PR DESCRIPTION
**Pull request by hannahbaumann**
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

[templates]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks
[input_validation]: https://github.com/OpenFreeEnergy/IndustryBenchmarks2024/tree/main/industry_benchmarks/utils/input_validation.py

### CODE REVIEW

File: docs/source/public/overview.rst [hunk 1]
```diff
@@ -733,12 +735,21 @@ To upload the data, please follow these steps:
...
+     * Title: For example something like: OpenFE 2024 Industry Benchmarking Project, Results from X (where X could be company name, or dataset name, or author.
+     * Creators: Please add as authors everyone involved in running the calculations.
+     * Description: Please report the OpenFE version that was used. You can get this information by running ``openfe --version`` in the commandline within the environment that you used to run the setup, simulations, and analysis of the data. In addition to that, please reference the `Industry Benchmark repo <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024>`_ as
+       it contains all necessary information about the simulation details.
+       E.g.: OpenFE 2024 Industry Benchmarking Project, Results from systems `jacs set; bace` and `merck set; shp2`. OpenFE version v1.0.1.  Simulation details can be found in the GitHub repository `https://github.com/OpenFreeEnergy/IndustryBenchmarks2024`
+
+  e) Add the compressed results folder (``results_data.zip``) to the zenodo upload. If multiple datasets are submitted in the same zenodo upload, please rename this file to reflect the benchmark system, e.g. `results_data_jacs_bace.zip`
```
Comment (IAlibay) [inline docs/source/public/overview.rst:751]: cc @jthorton - do we want to do this? i.e. will it be a problem for the analysis script?

**Review by IAlibay — CHANGES_REQUESTED** [review]
looks good to me, but I do have a question re: "multiple results in one entry".

File: docs/source/public/overview.rst [hunk 2]
```diff
@@ -733,12 +735,21 @@ To upload the data, please follow these steps:
...
-  b) Click the `New upload` button on the top right.
-  c) Fill out all the relevant information. In the `Description` section, please report the OpenFE version that was used. You can get this information by running ``openfe --version`` in the commandline within the environment that you used to run the setup, simulations, and analysis of the data. In addition to that, please reference the `Industry Benchmark repo <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024>`_ as
-     it contains all necessary information about the simulation details.
-  d) Add the compressed results folder (``results_data.zip``) to the zenodo upload.
-  e) Click the `Submit for review` button on the top right. We will then review the upload and publish it.
+  a) Login to zenodo or sign up if you do not have an account.
+  b) Upload the data to the **Open Free Energy Zenodo Community**. You can find the OpenFE community here: `https://zenodo.org/communities/openfreeenergy/records?q=&l=list&p=1&s=10&sort=newest <https://zenodo.org/communities/openfreeenergy/records?q=&l=list&p=1&s=10&sort=newest>`_.
+  c) Click the `New upload` button on the top right.
```
Comment (jthorton) [inline docs/source/public/overview.rst:740]:
```suggestion
  c) Click the ``New upload`` button on the top right.
```

File: docs/source/public/overview.rst [hunk 3]
```diff
@@ -733,12 +735,21 @@ To upload the data, please follow these steps:
...
+  d) Fill out all the relevant information.
+
+     * Digital Object Identifier: Click `No` and `Generate a DOI now!`
```
Comment (jthorton) [inline docs/source/public/overview.rst:743]:
```suggestion
     * Digital Object Identifier: Click ``No`` and ``Generate a DOI now!``
```

File: docs/source/public/overview.rst [hunk 4]
```diff
@@ -733,12 +735,21 @@ To upload the data, please follow these steps:
...
+     * Creators: Please add as authors everyone involved in running the calculations.
+     * Description: Please report the OpenFE version that was used. You can get this information by running ``openfe --version`` in the commandline within the environment that you used to run the setup, simulations, and analysis of the data. In addition to that, please reference the `Industry Benchmark repo <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024>`_ as
+       it contains all necessary information about the simulation details.
+       E.g.: OpenFE 2024 Industry Benchmarking Project, Results from systems `jacs set; bace` and `merck set; shp2`. OpenFE version v1.0.1.  Simulation details can be found in the GitHub repository `https://github.com/OpenFreeEnergy/IndustryBenchmarks2024`
+
+  e) Add the compressed results folder (``results_data.zip``) to the zenodo upload. If multiple datasets are submitted in the same zenodo upload, please rename this file to reflect the benchmark system, e.g. `results_data_jacs_bace.zip`
+  f) Click the `Submit for review` button on the top right. We will then review the upload and publish it.
```
Comment (jthorton) [inline docs/source/public/overview.rst:759]:
```suggestion
  f) Click the ``Submit for review`` button on the top right. We will then review the upload and publish it.
```

File: docs/source/public/overview.rst [hunk 5]
```diff
@@ -521,7 +521,9 @@ Gathering of results
 
 
 .. warning::
-  By default the gathering script does not replace ligand names! If you used proprietary names, please use the `--hide-ligand-names` flag.
+  By default the gathering script does not replace ligand names!
+  For the public datasets, we want to use the same ligand names as in the input files.
+  For the private datasets, if you used proprietary names, please use the `--hide-ligand-names` flag.
```
Comment (jthorton) [inline docs/source/public/overview.rst:526]:
```suggestion
  For the private datasets, if you used proprietary names, please use the ``--hide-ligand-names`` flag.
```

File: docs/source/public/overview.rst [hunk 6]
```diff
@@ -541,11 +541,11 @@ If the `fix network script <fix_networks>`_ was run, two alchemical networks nee
 
 If confidential ligand names were used, those can be replaced with generic ligand names. To do that you will need to add the flag ``--hide-ligand-names``:
 
-.. code-block:: bash
+thering.py --input_alchemical_network network_setup/alchemicalNetwork/alchemical_network.json --results-folder results_0 --results-folder results_1 --results-folder results_2 --hide-ligand-names
```
Comment (IAlibay) [inline docs/source/public/overview.rst:544]: Looks like an accidental formatting error got introduced here.

**Review by IAlibay — CHANGES_REQUESTED** [review]
Looks good, except the accidental formatting error 😅

File: docs/source/public/overview.rst [hunk 7]
```diff
@@ -731,14 +733,30 @@ To upload the data, please follow these steps:
...
-     it contains all necessary information about the simulation details.
-  d) Add the compressed results folder (``results_data.zip``) to the zenodo upload.
-  e) Click the `Submit for review` button on the top right. We will then review the upload and publish it.
+3. Create a directory for each dataset, naming it after the dataset, e.g. ``jacs_bace``. Then, move the  ``results_data.zip`` file into the respective dataset folder. Please make sure to include both the set name (e.g. jacs or waterset) and the target name (e.g. bace) since some targets occur in multiple sets.
+
+.. code-block:: bash
+
+    mkdir jacs_bace
+    mv results_data.zip jacs_bace
+
```
Comment (jthorton) [inline docs/source/public/overview.rst:742]: We should add something here about if its a private dataset make sure to copy in the experimental csv file into the folder.

File: docs/source/public/overview.rst (hunk 6 above)
Comment (hannahbaumann) [reply docs/source/public/overview.rst]: Oops, yep fixed it!

File: docs/source/public/overview.rst (hunk 7 above)
Comment (hannahbaumann) [reply docs/source/public/overview.rst:743]: Added this!

**Comment by hannahbaumann** [comment]
@jthorton @IAlibay : I think this is ready for a final review!

**Review by IAlibay — APPROVED** [review]

**Review by jthorton — APPROVED** [review]
LGTM!

**Pull request merged**